app/views/content_item_signups/confirm.html.erb

```diff
@@ -1,5 +1,4 @@
-<% page_heading = "Sign up to get emails" %>
-<% content_for :title, page_heading + ' - ' + @subscription.title %>
+<% content_for :title, t("content_item_signups.confirm.title") %>
 
 <% if live_content_item?(@content_item) %>
   <% content_for :back_link do %>
```

Comment:
I thought we were removing the back link on all these pages?

Reply:
Indeed we are. That will come in a separate PR, as we made the decision half way through the work.